types.ts

```diff
@@ -96,7 +96,7 @@ export interface NamedOptions {
 
   /**
    * A {@link OsRecord} containing the extensions for the respective
-   * operating-systems. By default this is `.so` for linux, `.dylib` for darwin
+   * operating-systems. By default this is `.so` for linux, freebsd, netbsd, aix, solaris and illumos, `.dylib` for darwin
```

Comment:
Have you checked that all these operating systems usually abide by those rules for dynamic libraries?

Reply:
yes